ex1_student_solution.py

```diff
@@ -75,8 +73,19 @@ def compute_forward_homography_slow(
         Returns:
             The forward homography of the source image to its destination.
         """
-        # return new_image
-        """INSERT YOUR CODE HERE"""
+
+        dst_image = np.zeros(dst_image_shape, dtype=np.uint8)
+        for i in range(src_image.shape[0]):
+            for j in range(src_image.shape[1]):
+                # there is a flip in the coordinates, as (row, column) = (y, x)
```

Comment:
why the flip?

Reply:
what I flip is the matrix indices, which gives (xs, ys, 1) since (i, j, 1) fits (ys, xs, 1) with numpy indexing
flipping back returns the coordinates to fit the matrix indices for the source value setting

Reply:
it's pretty annoying, I tried playing with the indices for quite a lot before getting something that makes sense (you can see in the results that the matching points reach where they're supposed to be in the destination coordinate system)